### ISSUE TITLE
Create download function for papers

### ISSUE THREAD

**Issue by ajburls** [dwyl/bestevidence#424]
As a user 
I would like to be able to download the citations from BestEvidence into a reference manager (ris, bib, enl).

(This request has come from user feedback)

This would need costing up.

**Comment by iteles** [comment]
@ajburls I believe this is a duplicate of #403 ?

**Comment by ajburls** [comment]
No, this is something  different - it is importing the results of a search into a reference manager. (Feedback suggested Rayyan) 

403 was about producing a BEAR
